src/app/services/movie.service.ts

```diff
@@ -14,11 +14,13 @@ export class MovieService {
     private http: Http) {
   }
 
-  public getMovies(searchValue?: string, page?: number): Observable<Movie[]> {
+  public getMovies(searchValue?: string, id?: number, page?: number): Observable<Movie[]> {
```

Comment:
Why do you changes this method? You have getDetails() for this

Reply:
You are right, this is superfluous. Has corrected.